.aws/src/main.ts

```diff
@@ -114,18 +116,65 @@ class ListAPI extends TerraformStack {
     });
   }
 
+  /**
+   * Creates a serverless aurora RDS.
+   * This function should only be used when the environment is Dev
+   * @private
+   */
+  private createRds(vpc: PocketVPC) {
+    return new ApplicationRDSCluster(this, 'dev-aurora', {
+      prefix: `${config.prefix}-v1`,
+      vpcId: vpc.vpc.id,
+      subnetIds: vpc.privateSubnetIds,
+      rdsConfig: {
+        databaseName: config.name.toLowerCase(),
+        masterUsername: 'pkt_listapi',
```

Comment:
Are the values for `databaseName`, `masterUsername` okay or should they be renamed to something more suitable?

Reply:
I think they're fine, but I'd put them in environment variables and add to the config since that's how we access them for the db connection

Reply:
I think they're fine, but I'd put them in environment variables and add to the config since that's how we access them for the db connection